chunky/src/java/se/llbit/chunky/renderer/Postprocess.java

```diff
@@ -36,6 +36,12 @@
     @Override public String toString() {
       return "Tonemap operator 1";
     }
+  },
+  TONEMAP2 {
+    @Override public String toString() { return "ACES filmic tone mapping"; }
```

Comment:
Whoops... My IDE collapsed the other blocks so they looked like one-liners.
I (and my IDE) would prefer to have the `@Override` in a separate line, and I've never really seen it on the same line as the method declaration, what do you think?